EIPS/eip-1559.md

```diff
@@ -192,7 +192,7 @@ class World(ABC):
 		else:
 			gas_used_delta = parent_gas_target - parent_gas_used
 			base_fee_per_gas_delta = parent_base_fee_per_gas * gas_used_delta // parent_gas_target // BASE_FEE_MAX_CHANGE_DENOMINATOR
-			expected_base_fee_per_gas = max(parent_base_fee_per_gas - base_fee_per_gas_delta, 0)
+			expected_base_fee_per_gas = parent_base_fee_per_gas - base_fee_per_gas_delta
```

Comment:
* `(parent_gas_target - parent_gas_used) / parent_gas_target` is always between 0 and 1 since `parent_gas_used < parent_gas_target` in this code path.
* This means that `base_fee_per_gas_delta` will be between `0/8` and `parent_base_fee_per_gas/8` in this code path.
* Which means `base_fee_per_gas_delta` will always be greater than 0.